pkg/ctl/create/cluster.go

```diff
@@ -112,14 +112,10 @@ func doCreateCluster(rc *cmdutils.ResourceCmd, params *createClusterCmdParams) e
 		cfg.Metadata.Version = api.DefaultVersion
 	}
 	if cfg.Metadata.Version != api.DefaultVersion {
-		validVersion := false
-		for _, v := range api.SupportedVersions() {
-			if cfg.Metadata.Version == v {
-				validVersion = true
-				break
+		if !isValidVersion(cfg.Metadata.Version) {
+			if isDeprecatedVersion(cfg.Metadata.Version) {
+				return fmt.Errorf("invalid version, %s is now deprecated, supported values: %s\nsee also: https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html", cfg.Metadata.Version, strings.Join(api.SupportedVersions(), ", "))
```

Comment:
What do you think about starting this error message with "unsupported version" instead of "invalid version"?
```suggestion
				return fmt.Errorf("unsupported version, %s is now deprecated, supported values: %s\nsee also: https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html", cfg.Metadata.Version, strings.Join(api.SupportedVersions(), ", "))
```

Reply:
I don't really mind. `invalid version, supported values: 1.11, 1.12, 1.13` was the generic error message up until now, hence I used `invalid`. 

Related to your suggestion, `invalid version, X.Y is no longer supported` was also suggested (and is being discussed in #1051), instead of `invalid version, X.Y is now deprecated`.

Unless a consensus can be found quickly and we can also close #1051 as part of this PR, I'd rather keep things as-is for now, and refine later after this has been bikeshedded properly 🙂.

Reply:
Sounds good :+1: